src/main/java/net/sf/jsqlparser/parser/ObjectNames.java

```diff
@@ -0,0 +1,31 @@
+/*-
+ * #%L
+ * JSQLParser library
+ * %%
+ * Copyright (C) 2004 - 2024 JSQLParser
+ * %%
+ * Dual licensed under GNU LGPL 2.1 or Apache License 2.0
+ * #L%
+ */
+package net.sf.jsqlparser.parser;
+
+import java.util.List;
+
+public class ObjectNames {
```

Comment:
This class is only used by the parser - it's the return value of the `RelObjectNames` production. I wasn't sure where to put it, as I don't think there are any other similar examples. Is it OK here?

Reply:
Put it into the parser grammar then.

Reply:
OK I'll add it as a static class in the parser